src/loose/state.js

```diff
@@ -158,4 +158,10 @@ export class LooseParser {
       plugin(this, pluginConfigs[name])
     }
   }
+
+  parse() {
+    let node = this.options.program || this.startNode()
+    this.next()
```

Comment:
Hmm, I believe they're different in parser/loose parser, but not completely sure as almost didn't touch the loose one. @marijnh should know better.